configs/train_tapecam_simulated_background.yaml

```diff
@@ -9,5 +9,6 @@ device_ids:
 
 # Dataset
 files:
-  background_fp: ""
+  dataset: Lensless/TapeCam-Mirflickr-Ambient
```

Comment:
yes because this dataset will also have to be aligned differently since it's not measured the same way as TapeCam. That means [alignment](https://github.com/LCAV/LenslessPiCam/blob/7dd98ced08a775255bbf238022655facbfdae0b4/configs/train_mirflickr_tape.yaml#L22) and [image_res](https://github.com/LCAV/LenslessPiCam/blob/7dd98ced08a775255bbf238022655facbfdae0b4/configs/train_mirflickr_tape.yaml#L17) need to be set differently in this new config. Image res was (600, 600) right? but alignment you have to determine like [this](https://colab.research.google.com/drive/1c6kUbiB5JO1vro0-IMd-YDDP1g7NFXv3#scrollTo=MtN7GWCIrBKr)